train/train_monolingual.py

```
@@ -0,0 +1,624 @@
+    )
+    n_params = -1
+    for dp, _, files in os.walk(model_dir):
+        for fname in files:
+            if fname.endswith(".safetensors"):
+                try:
+                    with safe_open(os.path.join(dp, fname), framework="np") as st:
+                        n_params = sum(
+                            int(np.prod(st.get_slice(k)[:].shape)) for k in st.keys()
+                        )
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)